features-json/av1.json

```diff
@@ -49,30 +49,30 @@
       "16":"n",
       "17":"n",
       "18":"n d #1",
-      "79":"y",
-      "80":"y",
-      "81":"y",
-      "83":"y",
-      "84":"y",
-      "85":"y",
-      "86":"y",
-      "87":"y",
-      "88":"y",
-      "89":"y",
-      "90":"y",
-      "91":"y",
-      "92":"y",
-      "93":"y",
-      "94":"y",
-      "95":"y",
-      "96":"y",
-      "97":"y",
-      "98":"y",
-      "99":"y",
-      "100":"y",
-      "101":"y",
-      "102":"y",
-      "103":"y"
+      "79":"n d #1",
```

Comment:
Since Edge is available on macOS and Linux as well, I'm not sure if I agree with `n d` (sticking `#1` on there is good in any case).

If it works on macOS and Linux out of the box, I'd argue for partial (=> `a`).

Reply:
Tried it macOS and didn't work there either.